### ISSUE TITLE
[a11y]: FileUploaderDropContainer component has critical violation "The tabbable element's role 'none' is not a widget role"

### ISSUE THREAD

**Issue by kenneth-nguyen1** [carbon-design-system/carbon#12498]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

7.56.0

### React version

17.0.2

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker - Latest Deployment

### Description

The `FileUploaderDropContainer` component has a critical a11y violation `"The tabbable element's role 'none' is not a widget role"` that was found in our product using the IBM Accessibility Checker tool and is re-producible on the [v10]( https://v7-react.carbondesignsystem.com/?path=/story/components-fileuploader--drag-and-drop-upload-container-example-application) and [v11](https://react.carbondesignsystem.com/?path=/story/components-fileuploader--drag-and-drop-upload-container-example-application) storybook as well.

v10 storybook
![FileUploader-v10](https://user-images.githubusercontent.com/114688923/199823607-bd2540df-b69d-461f-ae18-854477360454.png)

v11 storybook
![FileUploader-v11](https://user-images.githubusercontent.com/114688923/199823632-160b5594-a989-4a63-b4cb-becab44c88c7.png)

### Reproduction/example

Carbon v10 storybook - https://v7-react.carbondesignsystem.com/?path=/story/components-fileuploader--drag-and-drop-upload-container-example-application and/or Carbon v11 storybook - https://react.carbondesignsystem.com/?path=/story/components-fileuploader--drag-and-drop-upload-container-example-application

### Steps to reproduce

On the storybook link (either v10 or v11) for the FileUploader -> FileUploaderDropContainer story, run the IBM Accessibility Checker scan. The scan should report the critical violation mentioned above (scroll to the bottom of the violations list).

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by kenneth-nguyen1** [comment]
From looking at the component [code](https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/FileUploader/FileUploaderDropContainer.js#L117), maybe the fix is to move the `tabIndex` property from the `<label>` element to it's nested child `<div>` element? This potential fix would be the same for the v10 branch.

**Comment by kenneth-nguyen1** [comment]
Also, can the fix for this issue be backported to carbon v10 alongside v11? Thanks

**Comment by adarsh-kannamveetil** [comment]
Hi, we are also facing this issue, any update on this? Is this issue fixed in latest version?

**Comment by adarsh-ibm** [comment]
hi, am also facing this issue in one of our applications. Any update on this?

**Comment by nkn2022** [comment]
will this fix be available on v10?

**Comment by tw15egan** [comment]
We need to port this back to v10

**Comment by renato-souzaf** [comment]
Hi,
I'm facing this same issue when using `FileUploaderButton` component. Could you fix for `FileUploaderButton` component as well?

**Comment by tw15egan** [comment]
Closed via #14517, will be included in the next release of V10
